src/App/Fossa/Init/.fossa.yml

```diff
@@ -93,6 +93,44 @@ version: 3
 #     - test-project
 #
 #
+# # Configures settings for the release group you are interacting with through the FOSSA API.
+# releaseGroup:
```

Comment:
[minor] Let's include doc link for reference: https://docs.fossa.com/docs/release-groups